### ISSUE TITLE
Methods from base class are not called

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot#42506]
**Godot version:**

e2e20ae

**Issue description:**

Base.gd
```
extends Node2D
class_name Base

func _ready() -> void:
	print("ready")

func _process(delta: float) -> void:
	print("process")
```

Inherited.gd
```
extends Base
```

When you attach the Inherited.gd to a node, the methods are not called.

**Minimal reproduction project:**

[ReproductionProject.zip](https://github.com/godotengine/godot/files/5317978/ReproductionProject.zip)

**Comment by Calinou** [comment]
I guess you need to call `super()` on the methods for that to happen.

**Comment by CsloudX** [comment]
Can fix this first? I found so many times I need redefine methods and call super()

**Comment by KoBeWi** [comment]
Seems fixed in b76dfde
Except for `_init()`, see #50590
